src/components/molecules/Monaco/Monaco.tsx

```diff
@@ -329,11 +329,11 @@ const Monaco = (props: {editorHeight: string}) => {
   }, [selectedResourceId, resourceMap]);
 
   useEffect(() => {
-    if (editor) {
+    if (editor && code) {
```

Comment:
hm... isn't this going to set the selection every time the code changes (which is after each debounce) ? What happens if you are editing in a scrolled-down portion of the document?

Reply:
hmm, yeah! You are right I didn't think and test with the case you say.

Reply:
I think we need logic to detect when code changes from empty string for the first time and set the selection at the top, but not on every code change

Reply:
I did the similar approach